runtime/gc_glue_java/ScavengerRootClearer.cpp

```diff
@@ -245,7 +245,7 @@ MM_ScavengerRootClearer::scavengeContinuationObjects(MM_EnvironmentStandard *env
 							} else {
 								omrobjectptr_t forwardedPtr = forwardedHeader.getForwardedObject();
 								Assert_MM_true(NULL != forwardedPtr);
```

Comment:
Would it be better to use assertion with message and print `object` address?

Reply:
May be it make even more sense to replace assertion in line 242 to assertion with message and print `object` and `forwardedHeader`. There is common scenario we can get "bad" (most likely "stale") `next` pointer triggered assertion. In this case knowledge about location of such garbage might be useful for debugging